### ISSUE TITLE
Issue after adding btSimplePopUpDelegate

### ISSUE THREAD

**Issue by deepika1203** [balram3429/btSimplePopUp#9]
I am getting the following error `Use of undeclared type 'btSimplePopUP'` after adding `btSimplePopUpDelegate`

![screenshot 2015-11-13 19 40 32](https://cloud.githubusercontent.com/assets/15829511/11147887/35072854-8a3f-11e5-975c-fce984b3962d.png)

**Comment by balram3429** [comment]
Your typo fault..!!!!
Small p,
Not the capital P

**Comment by balram3429** [comment]
Hope this is resolved now.

**Comment by balram3429** [comment]
Hope you have added the bridging header for corresponding .h file in your swift project. ( refer here::http://stackoverflow.com/a/24034488)

**Comment by deepika1203** [comment]
Thanks for your response. I dont have access to my machine now. I will check it tomorrow and let you know.

**Comment by deepika1203** [comment]
BTW, where should I use small 'p' instead of 'P'. It is `btSimplePopUP` only know?

**Comment by deepika1203** [comment]
I do not think, it is a typo fault! I have added the bridging header also. And I dint type the whole class name. It is an auto completed text! So, some other issue may be there.

**Comment by balram3429** [comment]
Yes, I agree that the typo was not the mistake. That's why I have reopened the issue. Will check & update you on this.

**Comment by deepika1203** [comment]
Okay. Thank you..

**Comment by inceptiveKshitij** [comment]
Same issue for me also

**Comment by inceptiveKshitij** [comment]
please give a solution as early as possible
